.github/workflows/deploy-cf-it.yml

```diff
@@ -113,3 +113,38 @@ jobs:
         DEPLOY_CF_SA_KEY_JSON: ${{ secrets.DEPLOY_CF_SA_KEY_JSON }}
         CF_NAME: projects/${{ secrets.DEPLOY_CF_PROJECT_ID }}/locations/us-central1/functions/cf-http-json-${{ github.run_number }}
 
+  json-event-trigger:
+    name: test event trigger
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - id: build
+      name: Build dist
+      run: |-
+        npm install
+        npm run build
+    - id: deploy
+      uses: ./
+      with:
+        name: cf-event-json-${{ github.run_number }}
+        runtime: nodejs10
+        entry_point: helloWorld
+        source_dir: ./tests/test-node-func/
+        credentials: ${{ secrets.DEPLOY_CF_SA_KEY_JSON }}
+        event_trigger_type: providers/cloud.pubsub/eventTypes/topic.publish
+        event_trigger_resource: ${{ secrets.DEPLOY_CF_EVENT_PUBSUB_TOPIC }}
+    - uses: actions/setup-node@master
+      with:
+        node-version: 12.x
+    - run: npm install
+    - uses: google-github-actions/setup-gcloud@master # Set up ADC to make authenticated request to service
+      with:
+        service_account_email: ${{ secrets.DEPLOY_CF_SA_EMAIL }}
+        service_account_key: ${{ secrets.DEPLOY_CF_SA_KEY_JSON }}
+        export_default_credentials: true
+    - name: integration test clean up
```

Comment:
Are you missing the integration tests run?

Reply:
@averikitsch the `npm run e2e-tests` makes use of the  CF URL to make an HTTP request. 
https://github.com/google-github-actions/deploy-cloud-functions/blob/c9f24afdf1afe71a751e07499c0d007655f28d5f/tests/e2e/e2e.test.ts#L36

This new test creates a CF triggered via pubsub and cannot be tested via that method. Tracking creation of a new test for this here: https://github.com/google-github-actions/deploy-cloud-functions/issues/9